led_strip/examples/led_strip_spi_ws2812/main/led_strip_spi_ws2812_main.c

```diff
@@ -26,6 +34,7 @@ led_strip_handle_t configure_led(void)
         .led_pixel_format = LED_PIXEL_FORMAT_GRB, // Pixel format of your LED strip
         .led_model = LED_MODEL_WS2812,            // LED strip model
         .flags.invert_out = false,                // whether to invert the output signal
+        .config_pixel_order = config_order,
```

Comment:
Do you think it would be any more straightforward to pass a pointer to a const array with the list of indices, rather than a callback function?

IMO this would avoid the somewhat awkward API of `config_order`, where we are writing into an array of unknown size.